src/main/java/com/MedicalSpecializiation.java

```diff
@@ -1,9 +1,31 @@
 package com;
 
+import java.util.Scanner;
+
 public enum MedicalSpecializiation {
 
     PATHOLOGY,
     CARDIOLOGY,
     NEUROLOGY,
-    UROLOGY
+    UROLOGY;
+
+    public static MedicalSpecializiation getMedicialSpecializiation() {
+        System.out.println("Enter Doctor's medicial specializiation (PATHOLOGY, CARDIOLOGY, UROLOGY, NEUROLOGY):");
+        String selectedMedicialSpecializiation = getStringParameter("Your choise:");
+        if (selectedMedicialSpecializiation.equals(PATHOLOGY.name())) {
```

Comment:
You may iterate through list of values and return correct one.
https://stackoverflow.com/questions/1104975/a-for-loop-to-iterate-over-an-enum-in-java